packages/web-pkg/src/components/Spaces/ReadmeContentModal.vue

```diff
@@ -49,17 +49,6 @@ export default defineComponent({
       })
     }
   },
-  watch: {
-    'space.spaceQuota': {
-      handler: function (val) {
-        if (!val) {
-          return
-        }
-        this.setOptions()
-      },
-      deep: true
-    }
-  },
```

Comment:
Could you elaborate why you removed this?

Reply:
so probably just a copy&paste error right?